web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -551,7 +505,7 @@
   };
 </script>
 
-<svelte:window on:keydown={handleKeypress} />
+<svelte:window on:keydown={(event) => executeShortcuts(event, shortcuts)} />
```

Comment:
Why can't you use the other form here?

Reply:
Do you mean just a bunch of `use:shortcut` with other form? That's certainly possible, but I think using the list looks better with a lot of shortcuts.

Reply:
Oh, I think I misunderstood the interface for `use:shortcut`. I think it might be nicer if it basically took in a list of shortcuts (shortcut + callback) instead and then it looked the same everywhere. What do you think? Basically, why can't the default implementation for `use:shortcut` by `executeShortcuts`?

Reply:
I've added a new action called `use:shortcuts` which accept multiple shortcuts and replaces the `executeShortcuts` function. This means all shortcuts are registered via actions, which would make implementing the dynamic shortcuts modal easier

Reply:
Awesome, thanks.